command/connect/redirecttraffic/redirect_traffic_test.go

```diff
@@ -215,7 +214,7 @@ func TestGenerateConfigFromFlags(t *testing.T) {
 					},
 				},
 			},
-			expError: "failed parsing Proxy.Config: 1 error(s) decoding:\n\n* cannot parse 'bind_port' as int:",
```

Comment:
so there was a small change in how errors are reported that we needed to update here